### ISSUE TITLE
adding animation to title of the page 

### ISSUE THREAD

**Issue by gaur15** [nawed2611/paperbrain#61]
### Description

animation in "paperbrain" title to make it appealing

**Comment by SaadAnzar** [comment]
@gaur15 go ahead

**Comment by sagnik-p** [comment]
This issue has been inactive for a long time
I would like to work on this issue
Please assign

**Comment by SaadAnzar** [comment]
@gaur15 are you wroking on this?

**Comment by gaur15** [comment]
> @gaur15 are you wroking on this?
yes sir I'm working on this issue sorry for the delay. Can I have 1 more day only

**Comment by SaadAnzar** [comment]
@gaur15 yeah sure

**Comment by gaur15** [comment]
> @gaur15 yeah sure
sir I'm continuously  working on this issue  from 3, 4days but after making changes as I open  server it is crashing due to unmentioned file and so I'm not able to access front end .I thought I'm doing some error but I find out issue is in backend there is unmentioned file imported which is not present. 
as soon as this backend works I will complete my issue sorry again for the delay.

**Comment by sagnik-p** [comment]
@gaur15 
do you mind sharing your code here
thanks

**Comment by SaadAnzar** [comment]
> > @gaur15 yeah sure
> > sir I'm continuously  working on this issue  from 3, 4days but after making changes as I open  server it is crashing due to unmentioned file and so I'm not able to access front end .I thought I'm doing some error but I find out issue is in backend there is unmentioned file imported which is not present.
> > as soon as this backend works I will complete my issue sorry again for the delay.

Can you share screenshot of the error you are facing?

**Comment by gaur15** [comment]
> > > @gaur15 yeah sure
> > > sir I'm continuously  working on this issue  from 3, 4days but after making changes as I open  server it is crashing due to unmentioned file and so I'm not able to access front end .I thought I'm doing some error but I find out issue is in backend there is unmentioned file imported which is not present.
> > > as soon as this backend works I will complete my issue sorry again for the delay.
> 
> Can you share screenshot of the error you are facing?

<img width="960" alt="Screenshot 2023-06-18 124659" src="https://github.com/nawed2611/paperbrain/assets/133087361/d61ad038-9bb9-4de2-9567-c41abe201612">
index.js doesn't exist

**Comment by SaadAnzar** [comment]
@gaur15 bed directory is for database connection.
you can do this
cd .. // this will take you to root directory(paperbrain)
npm install
npm run dev
and it will work fine
